sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -267,50 +299,47 @@ def get_transceiver_info(self):
         ext_id = admin_info[consts.EXT_ID_FIELD]
         power_class = ext_id[consts.POWER_CLASS_FIELD]
         max_power = ext_id[consts.MAX_POWER_FIELD]
-
-        xcvr_info = {
+        self.xcvr_info =  copy.deepcopy(CmisApi.cmis_xcvr_info_dict)
+        self.xcvr_info.update({
```

Comment:
@vvolam Please add a testcase per our discussion to ensure that the length of the default dictionary is same as to the length of the dictionary returned by the get_transceiver_info function.